hackathon_site/dashboard/frontend/src/pages/Inventory/Inventory.tsx

```diff
@@ -1,53 +1,62 @@
-import React from "react";
+import React, { useEffect } from "react";
 import styles from "./Inventory.module.scss";
 import Drawer from "@material-ui/core/Drawer";
 import Header from "components/general/Header/Header";
 import Typography from "@material-ui/core/Typography";
-import TextField from "@material-ui/core/TextField";
 import Divider from "@material-ui/core/Divider";
 import IconButton from "@material-ui/core/IconButton";
 import Hidden from "@material-ui/core/Hidden";
 import Button from "@material-ui/core/Button";
 import RefreshIcon from "@material-ui/icons/Refresh";
-import SearchIcon from "@material-ui/icons/Search";
 import CloseIcon from "@material-ui/icons/Close";
-import InputAdornment from "@material-ui/core/InputAdornment";
 import FilterListIcon from "@material-ui/icons/FilterList";
-import Grid from "@material-ui/core/Grid";
-import EnhancedInventoryFilter from "components/inventory/InventoryFilter/InventoryFilter";
-import Item from "components/inventory/Item/Item";
+import InventoryFilter from "components/inventory/InventoryFilter/InventoryFilter";
+import InventoryGrid from "components/inventory/InventoryGrid/InventoryGrid";
 import ProductOverview from "components/inventory/ProductOverview/ProductOverview";
-import { productInformation, inventoryItems } from "testing/mockData";
+import { productInformation } from "testing/mockData";
+import InventorySearch from "components/inventory/InventorySearch/InventorySearch";
+import { RootState } from "slices/store";
+import { connect, ConnectedProps } from "react-redux";
+import { clearFilters, getHardwareWithFilters } from "slices/hardware/hardwareSlice";
```

Comment:
are we sorting imports by where they are imported from?

Reply:
Yeah, I should clean that up. I'd like imports to be:

- React import first
- Imports from libraries
- Imports of local components

Within each of those i don't feel like alphabetizing, but I should clean it up a bit. Will do that shortly.